pkgs/by-name/lo/logseq/package.nix

```
@@ -13,32 +14,43 @@
 }:
 
 stdenv.mkDerivation (finalAttrs: let
-  inherit (finalAttrs) pname version src appimageContents;
-
+  inherit (finalAttrs) pname version src;
+  inherit (stdenv.hostPlatform) system;
+  selectSystem = attrs: attrs.${system} or (throw "Unsupported system: ${system}");
+  suffix = selectSystem {
+    x86_64-linux = "linux-x64-${version}.AppImage";
+    x86_64-darwin = "drawin-x64-${version}.zip";
```

```suggestion
    x86_64-darwin = "darwin-x64-${version}.zip";
```

Is `drawin` on purpose?

---

My typo, which I keep making over and over, especially when I'm sleepy ;)